index.js

```diff
@@ -399,6 +402,35 @@
     function(x) { return $.FiniteNumber.test(x) && Number(x) !== 0; }
   );
 
+  //  Integer :: Type
+  $.Integer = $.NullaryType(
+    'sanctuary-def/Integer',
+    function(x) {
+      return $.ValidNumber.test(x) &&
+        Math.floor(x) === Number(x) &&
+        x >= MIN_SAFE_INTEGER  &&
+        x <= MAX_SAFE_INTEGER;
+    }
```

Comment:
I like to align the subexpressions when wrapping after `&&`:

```javascript
function(x) {
  return $.ValidNumber.test(x) &&
         Math.floor(x) === Number(x) &&
         x >= MIN_SAFE_INTEGER &&
         x <= MAX_SAFE_INTEGER;
}
```

Also, should we replace `Math.floor(x) === Number(x)` with `Math.floor(x) == x` do you think?